TGC/events/Germany VS Hungary Events.txt

```diff
@@ -0,0 +1,672 @@
+	desc = "EVTDESC2000000014"
+	picture = "german_rhine"
+
+	is_triggered_only = yes
+
+        option = {
+		name = "EVTOPTA2000000014"
+
+                set_global_flag = treaty_of_vienna_hungary
+
+		633 = { add_core = GER }
+		624 = { add_core = GER }
+		3286 = { add_core = GER }
+		623 = { add_core = GER }
+                3282 = { add_core = GER }
+                633 = {  secede_province = GER }
+		624 = {  secede_province = GER }
+		3286 = {  secede_province = GER }
+		623 = {  secede_province = GER }
+                3282 = {  secede_province = GER }
+                652 = { add_core = BAN }
+                653 = { add_core = BAN } 
+                3319 = { add_core = BAN }
+                
+                3309 = { add_core = BAN }
+                2538 = { add_core = BAN }
+                791 = { add_core = BAN }
+                792 = { add_core = BAN }
+                652 = {  secede_province = GER }
+                653 = {  secede_province = GER }
+                3319 = {  secede_province = GER }
+                
+                3309 = {  secede_province = GER }
+                2538 = {  secede_province = GER }
+                791 = {  secede_province = GER }
+                792 = {  secede_province = GER }
+                release = BAN
+                create_vassal = BAN
+                GER = { country_event = { id = 2000000002 days = 2 } }
+                Badboy = 2.5
+	}    
+}   
+
+##Hungary accepts Transylvania##
+
+country_event = {
+	id = 2000000015
+
+	title = "EVTNAME2000000015"
+	desc = "EVTDESC2000000015"
+	picture = "german_rhine"
+
+	is_triggered_only = yes
+        
+        option = {
+                name = "EVTOPTA2000000015"
+
+                set_global_flag = treaty_of_vienna_hungary
+
+		633 = { add_core = GER }
+		624 = { add_core = GER }
+		3286 = { add_core = GER }
+		623 = { add_core = GER }
+                3282 = { add_core = GER }
+                633 = {  secede_province = GER }
+		624 = {  secede_province = GER }
+		3286 = {  secede_province = GER }
+		623 = {  secede_province = GER }
+                3282 = {  secede_province = GER }
+                652 = { add_core = BAN }
+                653 = { add_core = BAN } 
+                3319 = { add_core = BAN }
+                
+                3309 = { add_core = BAN }
+                2538 = { add_core = BAN }
+                791 = { add_core = BAN }
+                792 = { add_core = BAN }
+                652 = {  secede_province = GER }
+                653 = {  secede_province = GER }
+                3319 = {  secede_province = GER }
+                
+                3309 = {  secede_province = GER }
+                2538 = {  secede_province = GER }
+                791 = {  secede_province = GER }
+                792 = {  secede_province = GER }
+                release = BAN
+                create_vassal = BAN
+                656 = {  secede_province = GER }
+                655 = {  secede_province = GER }

```

Comment:
Seceding to Germany is unnecessary, you can use HUN = { release = SIE } GER = { create_vassal = SIE } unless you want Transylvania to not have all of its cores.